main.js

```diff
@@ -21,4 +177,115 @@ navLink.addEventListener('click', () => {
     document.body.style.overflowY = 'scroll';
     menu.classList.toggle('show-menu');
   });
+
+ 
+
+  const cardWorks=document.querySelector('.card-works');
+  const topDesktop=document.createElement('div');
+  topDesktop.classList.add('top-desktop');
+  cardWorks.appendChild(topDesktop);
+
+  const headingPart=document.createElement('div');
+  headingPart.classList.add('heading-part');
+  topDesktop.appendChild(headingPart);
+
+  const headingOne=document.createElement('h1');
+  headingOne.textContent=ProjectDetail[1].title;
+  headingPart.appendChild(headingOne);
+
+  
+  const headingTwo=document.createElement('h2');
+  headingTwo.classList.add('desktopHeading')
+  headingTwo.textContent=ProjectDetail[1].popTitle;
+  headingPart.appendChild(headingTwo);
+
+  const closeIcon=document.createElement('p');
+  closeIcon.classList.add('fa-times');
+  closeIcon.classList.add('para');
+  // closeIcon.src=ProjectDetail[1].closeIconImage;
+  closeIcon.textContent="X";
+  headingPart.appendChild(closeIcon);
+  const closeInconActivity="document.getElementById('card-works').style.display=block";
+  closeIcon.onclick=closeInconActivity;
+
+  const technologiesSection=document.createElement('div');
+  technologiesSection.classList.add('technologies');
+  topDesktop.appendChild(technologiesSection);
+
+  const techList=document.createElement('ul');
+  technologiesSection.appendChild(techList);
+  for(let j=0;j<ProjectDetail[1].technologies.length; j+=1)
+  {
+      const techLi=document.createElement('li');
+      techList.appendChild(techLi);
+      techLi.classList.add('tec-list');
+      const techbtn=document.createElement('button');
+      techLi.appendChild(techbtn);
+      const classes= ['button'];
+      techbtn.classList.add(classes);
+      techbtn.textContent=ProjectDetail[1].technologies[j];
+
+      
+      
+  }
+
+  const snapshotDescriptionSource=document.createElement('div');
+  snapshotDescriptionSource.classList.add('snapshout-description-source');
+  cardWorks.appendChild(snapshotDescriptionSource);  
+  
+  const snapshotImage=document.createElement('div');
+  snapshotImage.classList.add('snapshot-image');
+  snapshotDescriptionSource.appendChild(snapshotImage);
+
+  const imagePlaceholder=document.createElement('img');
+  imagePlaceholder.classList.add('snapshot');
+  imagePlaceholder.src=ProjectDetail[1].imagePlaceholder;
+  imagePlaceholder.alt='';
+  snapshotImage.appendChild(imagePlaceholder);
+
+  const descriptionSource=document.createElement('div');
+  descriptionSource.classList.add('description-source');
+  snapshotDescriptionSource.appendChild(descriptionSource);
+  
+  const descriptions=document.createElement('div');
+  descriptions.classList.add('description');
+  descriptionSource.appendChild(descriptions);
```

Comment:
- [ ] as you noticed in the Figma design when you open your pop up window should also an overlay should be below your pop up window to blur the content of the portfolio  and also your pop up not centered correctly please try to fix that 

- [ ] Figma 
![image](https://user-images.githubusercontent.com/61790587/168919122-21392e7f-5d92-4c1f-84a8-1603def67f47.png)

- [ ] Yours
![image](https://user-images.githubusercontent.com/61790587/168919160-b4b5d2ac-386f-40a6-9075-1c1492fe2d70.png)